app/routes/company/components/CompaniesPage.js

```diff
@@ -3,65 +3,78 @@
 import React from 'react';
 import styles from './Company.css';
 import LoadingIndicator from 'app/components/LoadingIndicator';
+import InfiniteScroll from 'react-infinite-scroller';
 import { Link } from 'react-router';
 import { Image } from 'app/components/Image';
 import type { Company } from 'app/models';
-import { Button } from 'app/components/Form';
 
 type Props = {
   companies: Array<Company>,
+  fetchMore: () => void,
   showFetchMore: () => void,
-  fetchMore: () => void
+  hasMore: Boolean
 };
 
-export function CompanyItem({ company }: any) {
+const CompanyItem = function({ company }: any) {
```

Comment:
This component can be written as a arrow function like `CompanyList`.

Reply:
The `any` could also be replaced with `Company` to make it even more type safe :smile: